src/core/loss_detection.h

```diff
@@ -40,6 +40,18 @@ typedef struct QUIC_LOSS_DETECTION {
 
     uint32_t TimeOfLastPacketSent;
 
+    uint32_t LastAckedTime;
+
+    uint32_t LastAckedPacketSentTime;
+
+    uint32_t AdjustedLastAckedTime;
+
+    uint64_t TotalBytesSent;
+
+    uint64_t TotalBytesAcked;
+
+    uint64_t TotalBytesSentAtLastAck;
```

Comment:
I think these would be better placed right after `LargestAckEncryptLevel` above. Also, please add comments to these.